scale/job/models.py

```diff
@@ -470,6 +469,17 @@ def populate_job_data(self, job, data):
         job.disk_out_required = disk_out_required
         job.last_modified = modified
 
+        # Query JobInputFile model and if empty, populate the model
+        if not JobInputFile.objects.filter(job__id=job.id).values('input_file'):
+            job_inputs = []
+            job_input = JobInputFile()
+            job_input.job_id = job.id
+            for input_file in input_files:           
+                job_input.input_file_id = input_file.id
+                job_input.job_input = input_file.name
+                job_inputs.append(job_input)
+            JobInputFile.objects.bulk_create(job_inputs)
```

Comment:
You don't need the filter, there will never be anything in the job_input_file table until this function runs.
You need to create a new JobInputFile model for each input file.
The job_input.job_input field needs to be set to the input param name.